.github/ISSUE_TEMPLATE/bounty.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bounty
+about: Bounty
+title: ''
+labels: bounty
+assignees: ''
+
+---
+
+# Bounty
+
+## Scope
+<!-- A list of specific things which should be done to deliver the bounty. These could be seen as requirements to verify/review bounty against -->
+
+-
+-
+-
+
+## Deliverables
+<!-- Artifacts produced as the result of this bounty. Something that could be verified/reviewed. Some examples: updated code, deployment made, blog post published, public event conducted etc -->
+
+-
+-
+
+## Bounty Owner/Gardener
```

Comment:
@CtrlSushi I know you've been working on the bounty system using gitcoin - could you give this a quick read through and let me know if this preliminary process is ok. Thanks